apps/frontend/src/app/2_molecules/SystemStats/hooks/useGetAssetBalance.ts

```diff
@@ -17,14 +18,15 @@ import {
 import { getRskChainId } from '../../../../utils/chain';
 import { EcosystemDataType } from '../types';
 
+// todo: verify we need this, only used in one place
```

Comment:
Agree, there's also `useGetAssetBalanceByAddress()` which I already discussed refactoring with tiltom, it was a quick duplication to get something working as a hotfix - I think `apps\frontend\src\hooks\useAssetBalance.ts` should be close enough to replace these calls (with minor modifications)? I'll create another story for this update if so.